### PR TITLE
add missing docs, priority class, missing extra args

### LINKED ISSUE #6: nsqd docs and additional config
just starting convo here, will submit pr. Need to add docs for nsqd to readme. 

Additionally would like to add a couple of missing things to values.yaml for all nsq services: 

* priorityClassName: 
* extraArgs[]
* terminationMessagePolicy: FallbackToLogsOnError

### PR DESCRIPTION
**Pull request by Ashtonian**
closes #6

### CODE REVIEW

**Review by benjamin658 — CHANGES_REQUESTED** [review]
Correct the nsqd default replica.

File: beeinventor/nsq/README.md [hunk 1]
```diff
@@ -27,38 +27,61 @@ The following table lists the configurable parameters of the nsq chart and their
...
+| nsqadmin.affinity            | Affinity for pod assignment                                                          | `[]`                     |
+
+
+### nsqd
+
+| Value                   | Description                                                                          | Default              |
+|-------------------------|--------------------------------------------------------------------------------------|----------------------|
+| nsqd.enabled            | Enable the nsqd                                                                      | `true`               |
+| nsqd.fullnameOverride   | String to partially override nsq.nsqd.fullname                                       | `.Release.Name-nsqd` |
+| nsqd.replicaCount       | Number of nsqd replicas                                                              | `1`                  |
```
[the remark this answers is not in the document]
Comment (Ashtonian) [reply beeinventor/nsq/README.md]: fixed!

**Review by benjamin658 — APPROVED** [review]

**Comment by benjamin658** [comment]
Hi @Ashtonian , Let's bump the chart version to 1.1.6, and the PR will be good to go.

**Comment by LordNoteworthy** [comment]
Thank you for this !

**Comment by benjamin658** [comment]
> Thank you for this ! Let's get it merged :)

Waiting for @Ashtonian bump chart version in this PR.

**Comment by benjamin658** [comment]
I have created a new commit d9aa2e92ba763d2f2a7e2a2edde19a2fc7d01bd0 to bump the chart version to 1.1.6.

Thanks for your contribution!

**Comment by benjamin658** [comment]
@LordNoteworthy ,  extraArgs is supported in the new release.

The tarball has been committed accidentally in 1.1.6, so you better use the 1.1.7.

**Comment by LordNoteworthy** [comment]
Thank you so much ! Much appreciated <3

**Comment by LordNoteworthy** [comment]
Probably we missed something:

`{{ toYaml .Values.nsqadmin.extraArgs | indent 8 }}` should be `{{*-* toYaml .Values.nsqadmin.extraArgs | indent 8 }}`

Otherwise, we end up like this:

image

**Pull request merged**